### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by loic425** [comment]
@lchrusciel What do you think about these fixes?

File: composer.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
         "symfony/validator": "^4.4|^5.0",
         "symfony/yaml": "^4.4|^5.0",
         "willdurand/hateoas-bundle": "^2.0",
-        "winzou/state-machine-bundle": "^0.3|^0.4"
+        "winzou/state-machine-bundle": "^0.4.3"
```
Comment (lchrusciel) [inline composer.json:42]: Why is it needed?

**Review by lchrusciel — COMMENTED** [review]
Hey Loic,

thanks for your work and fixes seem ok. However, I'm wondering why do we need to bump the state machine version? Not a big deal, but we are using `0.3` in Sylius right now.

File: composer.json (hunk 1 above)
Comment (loic425) [reply composer.json:42]: There's a bug on winzou with dependencies between state-machine and state-machine bundle. Maybe there's something better to do:
"winzou/state-machine-bundle": "^0.3.2|^0.4.3"
I'm gonna try that.

Comment (loic425) [reply composer.json]: @lchrusciel But I think we have to improve CI configuration to test these two state machine versions.

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thanks, Loïc! :1st_place_medal:

**Pull request merged**